lib/active_model_serializers/logging.rb

```diff
@@ -104,8 +104,10 @@ def logger_tagged_by_active_model_serializers?
 
     class LogSubscriber < ActiveSupport::LogSubscriber
       def render(event)
+        serializer = event.payload[:serializer]
+        return if serializer == ActiveModel::Serializer::Null
+
         info do
-          serializer = event.payload[:serializer]
```

Comment:
I could see people disagreeing with this behavior.. not logging at all when logger is null... thoughts?